### PR TITLE
Leyi Kang HW01

### PR DESCRIPTION
**Pull request by lindsaykkk**
I basically followed the steps introduced in the "Homework Workflow" video on the CFSS website. After I cloned the README file into the RStudio on my laptop, I wrote a short bio and practiced some Markdown functions such as making headers, emphasis, lists, and importing an image of the sea and a link to the lego website. Since this is the first time I used R, I took some time getting familiar with the R Markdown and read the help guide of Markdown on RStudio webpage. I successfully inserted the image and the link the first time, but still tried to improve by committing for four times, each making a little changes. Overall, I didn't think it's hard to finish the assignment after I had a clear idea about how to use GitHub and R Markdown.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
...
+*This sentence will be italic.* or _This will also be italic._
+
+**This sentence will be bold.** or __This will also be bold.__
+
+__This will be *both*.__
+
+* Unordered List
+* Colors
+  + blue
+  + red
```
Comment (jcpruin) [inline README.md:23]: I think I recognize this bit from one of the reference articles? I think we were supposed to use the extra text options (headers, italics, etc.) to continue with our introduction.

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+  + yellow
+  
+1. Ordered list
+2. Animals
+   1. horse
+   2. dolphin 
+
+
+Here's a picture of the sea: ![Sea](https://techcrunch.com/wp-content/uploads/2017/05/gettyimages-580833893.jpg?w=990&crop=1)
```
Comment (jcpruin) [inline README.md:32]: I love how you got the image function to work! I couldn't figure it out for the life of me.

File: README.md (hunk 1 above)
Comment (djanvier) [reply README.md:23]: That's correct! There seemed to be some confusion here, but that's alright for now.

File: README.md (hunk 2 above)
Comment (djanvier) [reply README.md:32]: Beautiful picture!

File: README.md [hunk 3]
```diff
@@ -1 +1,36 @@
...
+
+
+[lego](https://www.lego.com/en-us)
```
Comment (djanvier) [inline README.md:35]: 😮

**Review by djanvier — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |        x      |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |           |         x     |            |
| **Achievement, creativity** |           |     x         |            |
| **Reproducibility**         |    x       |              |            |

- Nice submission! Although it wasn't exactly what we asked for, I'm glad that you were able to practice a multitude of features using Markdown syntax
- If you ever have any clarifying questions or concerns about future homework assignments, feel free to shoot me an email!

**Pull request closed without merging**